lib/noble.js

```diff
@@ -59,22 +79,41 @@ module.exports = {
   },
   publicKeyConvert: function (output, pubkey) {
     try {
-      output.set(secp256k1.pointCompress(pubkey, isCompressedPublicKey(output)))
+      output.set(bitcoinerlab.pointCompress(pubkey, isCompressedPublicKey(output)))
       return 0
     } catch (e) {
       if (e.message === errors.EXPECTED_POINT) return 1
       return 2
     }
   },
   publicKeyNegate: function (output, pubkey) {
-    return elliptic.publicKeyNegate(output, pubkey)
+    try {
+      output.set(noble.Point.fromHex(pubkey).negate().toRawBytes(isCompressedPublicKey(output)))
+    } catch (e) {
+      return 1
+    }
+    return 0
   },
   publicKeyCombine: function (output, pubkeys) {
-    return elliptic.publicKeyCombine(output, pubkeys)
+    try {
+      let point = noble.Point.fromHex(pubkeys[0])
```

Comment:
This code assumes `pubkeys` is an array of hex strings. But the original code in `elliptic.publicKeyCombine` assumes `pubkeys` is an array of `UInt8Array`s. Which is it?

Reply:
Noble handles both arraylikes and strings

Reply:
```ts
declare type Hex = Uint8Array | string;
```

Reply:
Ah nice!